### PR TITLE
Update seqtk to version 1.3 and allow fasta/q gz input and ouput

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

File: tools/seqtk/macros.xml [hunk 1]
```diff
@@ -18,11 +18,14 @@
         </stdio>
     </xml>
     <xml name="in_fq">
-        <param name="in_file" type="data" format="fastq" label="Input FASTQ file"/>
+        <param name="in_file" type="data" format="fastq.gz" label="Input FASTQ file"/>
```
Comment (abretaud) [inline tools/seqtk/macros.xml]: did you mean `fastq,fastq.gz`?

Comment (mvdbeek) [reply tools/seqtk/macros.xml]: I did ...

Comment (mvdbeek) [reply tools/seqtk/macros.xml]: rebased with the fix

**Comment by abretaud** [comment]
Thanks @mvdbeek!

**Pull request merged**